docs/telemetry.md

```diff
@@ -0,0 +1,43 @@
+# Telemetry
+
+Coder collects telemetry data from all free installations. Our users have the right to know what we collect, why we collect it, and how we use the data.
+
+## What we collect
+
+First of all, we do not collect any information that could threaten the security of your
+installation. For example, we do not collect parameters,
+environment variables, or passwords.
+
+You can find a full list of the data we collect in the source code [here](https://github.com/coder/coder/blob/main/coderd/telemetry/telemetry.go).
+
+We offer three levels of telemetry that can be configured with the
+`CODER_TELEMETRY=x` environment variable. For example, telemetry level 1 can
+be configured with `CODER_TELEMETRY=1`.
```

Comment:
Let's adjust these docs to describe the current behavior, and make an issue if we want to change it.

Right now `CODER_TELEMETRY` can be set to `true` or `false`, and defaults to `true`.

Reply:
> Let's adjust these docs to describe the current behavior, and make an issue if we want to change it.

👍